client/src/components/event-types/EventTypesList.vue

```diff
@@ -3,12 +3,14 @@
         <div class="row ma-0">
             <VCol cols="12" sm="12" md="3" class="pa-0">
                 <VTextField
+                    class="search-input"
                     type="text"
                     :label="lang.SEARCH"
                     prepend-inner-icon="mdi-magnify"
                     dense
                     filled
-                    solo-inverted
+                    solo
+                    background-color="rgba(224, 224, 224, 0.3)"
```

Comment:
This hard coded constant in code maybe your can use colors from theme. And in code below you have v-model we have be advised to avoid of using it.